docs/data/charts/lines/ConnectNulls.tsx

```diff
@@ -31,6 +31,7 @@ export default function ConnectNulls() {
         ]}
         height={200}
         margin={{ top: 10, bottom: 20 }}
+        skipAnimation
```

Comment:
The `animation` and `connectNulls` do not seem to play nicely together. 🙈 
WDYT, would it make sense to call this thing out in the docs? 🤔

Reply:
The issue is toggling between `connectNulls` set to `true` and `false` which, except in a documentation about this feature make no sens. Maybe on toolpad to let users configure their chart 🤔

But I don't think It's worth mentionning

Reply:
Ok, fair point. We can disregard this comment. 👌